storage/acl.go

```diff
@@ -20,7 +20,7 @@ import (
 	"reflect"
 
 	"cloud.google.com/go/internal/trace"
-	storagepb "cloud.google.com/go/storage/internal/apiv2/stubs"
+	storagepb "cloud.google.com/go/storage/internal/apiv2/storagepb"
```

Comment:
I guess this no longer needs an alias import, is there some other reason you are keeping the alias?